src/js/components/Investigation/FindingCard.js

```diff
@@ -66,7 +61,7 @@ export default React.memo<Props>(function FindingCard({finding}: Props) {
     )
   }
 
-  const template = [
+  const menu = usePopupMenu([
```

Comment:
Changed the return value of usePopMenu to include an "onClick" handler along with some other nice things like "isOpen".